### PR TITLE
Homework: NWD NWW

### PR DESCRIPTION
**Pull request by michalsoloducha**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @michalsoloducha

**Comment by ziobron** [comment]
⏱ 3 XP extra for @michalsoloducha for on-time delivery

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,14 @@
 #pragma once
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    if(rhs != 0) return NWD(rhs, lhs%rhs);
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:4]: one command one line

Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:4]: Add space between operations like '+' '-' '!=' '==' '%' etc...

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
+    if (lhs < 0) lhs = -lhs;
+    return lhs;
 }
 
 int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    if (lhs == 0 && rhs ==0) return 0;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:10]: one command one line

File: homework/nwd-nnw/nwdNww.hpp [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
+    int result = lhs/NWD(lhs, rhs)*rhs;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:11]: Add space between operations like '+' '-' '!=' '==' '%' etc...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**